photon-core/src/main/java/org/photonvision/vision/apriltag/AprilTagDetectorParams.java

```diff
@@ -30,7 +30,7 @@
 
 public class AprilTagDetectorParams {
     public static AprilTagDetectorParams DEFAULT_36H11 =
```

Comment:
Should this say `DEFAULT_36H11` as the variable name when it is `kTag16h5` as the set tag family?